docs/use-cases/use_case_specifications.md

```
@@ -0,0 +1,272 @@
+---
+
+## UC2. Add Comment (코멘트 추가)
+
+| 항목 | 내용 |
+|---|---|
+| ID | UC2 |
+| 이름 | Add Comment |
+| 범위 | Issue Tracking System |
+| 수준 | Subfunction |
+| Primary Actor | Auth User(Tester, Dev, PL) |
+| Stakeholders & Interests | - Auth User: 이슈에 자신의 의견과 진행 상황 등을 기록하고 싶다<br>- 조직: 이슈 배정 이후 해당 이슈와 관련된 사용자들의 이슈에 대한 
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)